dev/cdn/build/nativeNodeModules.js

```diff
@@ -0,0 +1,37 @@
+const nativeNodeModulesPlugin = {
```

Comment:
So this is a custom esbuild plugin? Where did you get an overview on writing this?

Reply:
Yes, this is a custom node modules resolver. Esbuild is still in a pretty hacky state at times, lots of folks just port over a webpack-esque thing to esbuild compatibility.

I took it from a GitHub issue from one of the esbuild maintainers when building pasteboard a few months back, I'm not sure where it's from at this point.